doc-build/md051.md

```diff
@@ -38,6 +38,14 @@ attribute can be used to define a fragment:
 An `a` tag can be useful in scenarios where a heading is not appropriate or for
 control over the text of the fragment identifier.
 
+GitHub also supports links to [specific lines][github-lines-links].
+
+For example, to link to line 12 of current Markdown file:
```

Comment:
I'm not sure I want to suggest doing this because GitHub also says you need to do `plain=1`. I'd say let's say something like "That syntax looks like this is practice:".